package.json

```
@@ -1,64 +1,70 @@
 {
   "name": "eslint.github.io",
-  "version": "1.0.0",
   "private": true,
+  "version": "1.0.0",
   "description": "This contains the code running on eslint.org.",
-  "main": "index.js",
-  "directories": {
-    "doc": "docs"
-  },
-  "engines": {
-    "node": ">=8.0.0"
-  },
-  "scripts": {
-    "build:jekyll": "bundle exec jekyll build",
-    "build:webpack": "webpack --mode=production",
-    "build:less": "lessc src/styles/main.less assets/build/styles/main.css --clean-css --source-map=assets/build/styles/main.css.map",
-    "build": "npm run build:less && npm run build:webpack && npm run build:jekyll",
-    "start:jekyll": "bundle exec jekyll serve --config _config.yml,_config.dev.yml",
-    "start:webpack": "webpack -w --mode=development",
-    "start:docker": "npm run build:less && docker run --rm -v `pwd`:/srv/jekyll -p 4000:4000 -it jekyll/jekyll:3.8 jekyll serve -V --config _config.yml,_config.dev.yml",
-    "start": "npm run build:less && npm run start:jekyll",
-    "lint": "eslint --ext=.js,.jsx .",
-    "lint:fix": "npm run lint -- --fix",
-    "test": "npm run lint"
-  },
+  "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/eslint/eslint.github.io.git"
   },
-  "author": "",
-  "license": "MIT",
   "bugs": {
     "url": "https://github.com/eslint/eslint.github.io/issues"
   },
   "homepage": "https://github.com/eslint/eslint.github.io#readme",
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "scripts": {
+    "build:eleventy": "npm run convert-yaml && eleventy",
+    "build:webpack": "webpack --mode=production",
+    "build:less": "lessc src/styles/main.less _site/assets/styles/main.css --clean-css --source-map=_site/assets/styles/main.css.map",
+    "build": "npm run build:eleventy && npm run build:less && npm run build:webpack",
+    "start:eleventy": "npm run convert-yaml && NODE_ENV=development eleventy --watch",
+    "start:webpack": "webpack -w --mode=development",
+    "start": "node _tools/dev-server.js",
+    "lint": "eslint --ext=.js,.jsx .",
+    "test": "npm run lint",
+    "fix": "npm run lint -- --fix",
+    "convert-yaml": "node _tools/yaml-to-json.js '_data/*.yml'"
```

```suggestion
    "convert-yaml": "node _tools/yaml-to-json.js \"_data/*.yml\""
```

In cmd.exe, single quotes are not a special character to enclose strings.

---

I was just debugging this and noticed the same thing! Will fix.